resources/views/plugins.blade.php

```diff
@@ -1,5 +1,5 @@
-@foreach(config('adminlte.plugins') as $plugin)
-    @if($plugin['active'] || View::getSection('plugins.' . $plugin['name']))
+@foreach(config('adminlte.plugins') as $pluginName => $plugin)
+    @if($plugin['active'] || View::getSection('plugins.' . $plugin['name'] ?? $pluginName))
```

Comment:
Just for safety, can you replace `$plugin['name'] ?? $pluginName` by `($plugin['name'] ?? $pluginName)` ??

Reply:
Sure.
Done.